backend/organization/serializers/project.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+
+from organization.models import Project
+from climateconnect_api.serializers.common import SkillSerializer
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+    skills = SkillSerializer(many=True)
+
```

Comment:
We'll also display organization members. This could be included here, but could also do done in a seperate route. If we want to do it in a seperate route, we should add an issue for that.

Reply:
Organization members or Project members? We have two tables. I would prefer it to be a new API. It will make backend response faster I believe.

Reply:
Oh I meant to say project members. Sounds good!

Reply:
@positiveimpact Is organization members API will also be open for everyone (i.e.: permission=allowany)?